draft-ietf-quic-qpack.md

```diff
@@ -975,6 +984,21 @@ The entries in the following table are registered by this document.
 | QPACK Decoder Stream         | 0x68   | {{wire-format}}           | Both   |
 | ---------------------------- | ------ | ------------------------- | ------ |
 
+## Error Code Registration
+
+This document establishes one new error code in the "HTTP/QUIC Error Code"
+registry established in {{QUIC-HTTP}}.
+
+Name:
+: HTTP_QPACK_DECOMPRESSION_FAILED
+
+Code:
+: 0x06
+
+Description:
+: QPACK failed to decompress a frame and cannot continue.
```

Comment:
I'm not sure 'frame' is the right term here.  This is basically any decoding error of any kind, which may occur outside of a frame.

Reply:
Good point -- just stale text.